mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -357,6 +357,7 @@ CLASS aia net/minecraft/entity/Entity
 		ARG 3 y
 		ARG 5 z
 	METHOD h getYaw (F)F
+		ARG 1 partial
```

Comment:
I think the standard Fabric name is `tickDelta`

Reply:
So should I change all the `partial` parameters in `MathHelper` to `tickDelta`

Reply:
I don't think so -- they may not specifically be used in the context of tick interpolation.

Reply:
I'd change MathHelper's to delta over partial, and keep using tickDelta/delta